app/models/service_template_transformation_plan_task.rb

```diff
@@ -233,6 +233,16 @@ def get_conversion_state
     update_options(updates)
   end
 
+  # Applies the limits for the task.
+  # CPU and network limits are set at the host level but other limits may not.
+  def apply_virtv2v_limits
+    limits = {
+      :cpu     => conversion_host.get_cpu_limit,
+      :network => conversion_host.get_network_limit
+    }
+    conversion_host.apply_virtv2v_limits(options[:virtv2v_wrapper]['throttling_file'], limits)
```

Comment:
This construction of `limits` seems redundant. If this `self` object has no opinion on how the limits are constructed, why not let `conversion_host` do it internally?

Reply:
I think I kind of agree with @jameswnl. Seems like this could be refactored away or delegated.